src/main/java/com/bootcamp/demo/model/Booking.java

```diff
@@ -0,0 +1,104 @@
+package com.bootcamp.demo.model;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public final class Booking {
+    private String id;
+    private String serialNumber;
+    private long accountId;
+    private String startDate;
+    private String endDate;
+    private PaymentStatus payment;
+
+    public Booking() {
+    }
+
+    public Booking(String serialNumber, long accountId, String startDate, String endDate, PaymentStatus paymentStatus) {
+        this.id = UUID.randomUUID().toString().replace("-", "");
```

Comment:
No need to replace "-" with empty. You could simply use the string value of an uuid.